robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -184,7 +184,6 @@ def execute(self, userdata=None):
             self.robot.speech.speak('I am sorry but I cannot move my arm to the object position', block=False)
             rospy.logerr('Grasp failed')
             arm.reset()
-            arm.send_gripper_goal('close', timeout=0.0)
```

Comment:
First this line was redundant, however now, since the reset function is changed, removing this line changes the behaviour of the state. Since if this if-statement holds a 'failed' is returned, which leads to the 'resetonFailure' state, in which this different behaviour seems to work out fine.